### ISSUE TITLE
hocr-lines outputs byte strings in python3

### ISSUE THREAD

**Issue by kba** [tmbdev/hocr-tools#86]
E.g. `./hocr-lines test/testdata/sample.html`

```python
b'1 Down the Rabbit-Hole'
b'Alice was beginning to get very tired of sitting by her sister on the bank,'
b'and of having nothing to do: once or twice she had peeped into the book her'
b'sister was reading, but it had no pictures or conversations in it, `and what is'
b"the use of a book,' thought Alice `without pictures or conversation?"
```

**Comment by zuphilip** [comment]
Ah, I see. Also the test should be better to catch this case, see https://github.com/tmbdev/hocr-tools/pull/84#discussion_r83638952 .

@kba Are you working on this?

**Comment by kba** [comment]
Yes.

**Comment by zuphilip** [comment]
This is fixed by #87.
